### DISCUSSION TITLE
Target solutions not saved from one pc to another?

### DISCUSSION THREAD

**Discussion by blkhwks19** [cssbattle/forum#314] [General]
I've been doing these targets for a while, and recently got a new computer. Logged in just fine on the new pc but can't see my past solutions? I can still log in on the old pc and see all the past solutions, so they're definitely still there, why won't the new pc load them?

**Comment by chinchang** [comment]
If you are logged in, you can see your past solutions through the "My Solutions" button and load back any of them.
![image](https://user-images.githubusercontent.com/379918/193392701-dfe5979e-96a2-4850-aa93-400e3e6501eb.png)
